alf/algorithms/actor_critic_loss.py

```diff
@@ -111,13 +111,18 @@ def __call__(self, training_info: TrainingInfo, value):
         returns, advantages = self._calc_returns_and_advantages(
             training_info, value)
 
-        if self._debug_summaries and common.should_record_summaries():
+        def _summary():
```

Comment:
can we use something like the following, which is more readable:
```python
if tf.logical_and(self._debug_summaries, common.should_record_summaries()):
```

Reply:
This won't work as I tried.

Reply:
> This won't work as I tried.
I've tried it before and it did not work, but it works now , I can't remember what the problem is.

Reply:
If I run ac_cart_pole.gin, it will crash.

Reply:
> If I run ac_cart_pole.gin, it will crash.

yes , it will not work 
i saw it as `tf.cond(tf.logical_and(self._debug_summaries, common.should_record_summaries()) ...`  before ...